genai_factory/src/genai_factory/__main__.py

```diff
@@ -89,16 +89,27 @@ def run(
         elif _IS_LOCAL_CONFIG:
             config = WorkflowServerConfig.local_config()
         else:
-            config = WorkflowServerConfig()
-
-    # Set the configuration:
-    workflow_server.set_config(config=config)
+            config = None  # The user count on the default config, or he set it manually in the script.
 
     # Importing the workflows server instance:
     click.echo(f"Importing workflows server from: {workflows_path}")
 
+    # import workflow_server from given path:
+
+    # Load the module from the file path
+    spec = importlib.util.spec_from_file_location("workflow_server", workflows_path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
```

Comment:
This method is no longer optional when [exec_module()](https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.exec_module) is defined.